vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_authorization_unittest.cc

```diff
@@ -0,0 +1,300 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <tuple>
+#include <utility>
+
+#include "base/test/task_environment.h"
+#include "bat/ledger/internal/database/database_mock.h"
+#include "bat/ledger/internal/ledger_client_mock.h"
+#include "bat/ledger/internal/ledger_impl_mock.h"
+#include "bat/ledger/internal/state/state_keys.h"
+#include "bat/ledger/internal/uphold/uphold_authorization.h"
+#include "bat/ledger/option_keys.h"

```

Comment:
Same as above; this include may not be needed after BAP was removed

Reply:
Resolved in 829eadc955d034cd2c4bf783bfc0e9455a73ea1a.